client/signup/steps/design-type-with-atomic-store/index.jsx

```diff
@@ -90,13 +97,24 @@ class DesignTypeWithAtomicStoreStep extends Component {
 	};
 
 	handleNextStep = designType => {
+		if ( ! this.props.countryCode ) {
```

Comment:
What if user clicked on something else than `store` design type? We don't want to differentiate between US/CA users and other parts of world there.

Reply:
Fixed: if the user clicks on something else than "store", I don't wait for the `countryCode` and proceed immediately.